theano/tests/test_flake8.py

```diff
@@ -72,14 +72,6 @@
     "tensor/tests/test_blas_scipy.py",
     "tensor/tests/test_mpi.py",
     "tensor/nnet/__init__.py",
-    "tensor/nnet/tests/__init__.py",
```

Comment:
Leave the `__init__` file in the whitelist.  It's too hard to clean up properly.